src/Infrastructure/Repository/SettingRepository.php

```diff
@@ -170,4 +189,69 @@ public function saveMultiple(array $settings): array
 
         return $settings;
     }
+
+    /**
+     * @return array
+     */
+    public function getSettingDefinition(): array
```

Comment:
Does it make sense to add this public method to the interface this class implements?

Reply:
No, it must be encapsulated. only one method will create an entity by definition.